openflow15/nxt_message.go

```diff
@@ -775,14 +773,17 @@ type PacketIn2PropPacket struct {
 }
 
 func (p *PacketIn2PropPacket) Len() (n uint16) {
-	n = p.PropHeader.Len() + p.Packet.Len()
+	return uint16(p.actLength())
+}
 
+func (p *PacketIn2PropPacket) actLength() int {
```

Comment:
s/actLength/actualLength
no need to abbreviate here

Reply:
updated.